django-stubs/db/models/fields/related_descriptors.pyi

```diff
@@ -86,6 +86,18 @@ class ReverseManyToOneDescriptor:
     def __get__(self, instance: Model, cls: Any = ...) -> type[RelatedManager[Any]]: ...
     def __set__(self, instance: Any, value: Any) -> NoReturn: ...
 
+class RelatedManager(BaseManager[_M], Generic[_M]):
```

Comment:
I think a comment to the fact that this doesn't directly correspond to a runtime class is useful. Feel free to tweak.

```suggestion
# Fake class, Django defines RelatedManager inside a function body
class RelatedManager(BaseManager[_M], Generic[_M]):
```

Would make sense to add the same for `ManyRelatedManager` as well

Reply:
Am thinking to copy the comment from the `allowlist.txt` where these classes are also present

> # 'RelatedManager' does exist and is declared locally, inside a function body

As well as

> # 'ManyRelatedManager' does exist and is declared locally, inside a function body

Reply:
Nvm, that didn't sound so good. Your suggestion reads better